client/my-sites/importer/author-mapping-pane.jsx

```diff
@@ -81,14 +79,13 @@ class AuthorMappingPane extends PureComponent {
 			return this.props.translate(
 				'There are multiple authors on your site. ' +
```

Comment:
Maybe use "original" here for more specificity, especially in the case of WordPress sites.
```suggestion
				'There are multiple authors on your original site. ' +
```